### PR TITLE
(DOCSP-28353): Pagination with Web SDK + GraphQL API

### PR DESCRIPTION
**Pull request by mongodben**
## Pull Request Info

Pagination with Web SDK + GraphQL API

### Jira

- https://jira.mongodb.org/browse/DOCSP-28353

### Staged Changes

- [Apollo/React Guide](https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-28353/web/graphql-apollo-react/)

### Reminder Checklist

If your PR modifies the docs, you might need to also update some corresponding
pages. Check if completed or N/A.

- [x] Create Jira ticket for corresponding docs-app-services update(s), if any
- [x] Checked/updated Admin API
- [x] Checked/updated CLI reference

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: examples/web/src/GraphQLPagination.jsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+  }
+
+  function goToPrevPage() {
+    setVariables({
+      nextTitle: data.movies[0].title,
+      prevTitle: undefined,
+      limit: resultsPerPage,
+      sortDirection: "TITLE_DESC",
+    });
+  }
```
Comment (mongodben) [inline examples/web/src/GraphQLPagination.jsx:94]:
```suggestion
  }

```

File: source/examples/generated/web/GraphQLPagination.snippet.paginate.jsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  }
+
+  function goToPrevPage() {
+    setVariables({
+      nextTitle: data.movies[0].title,
+      prevTitle: undefined,
+      limit: resultsPerPage,
+      sortDirection: "TITLE_DESC",
+    });
+  }
```
Comment (mongodben) [inline source/examples/generated/web/GraphQLPagination.snippet.paginate.jsx:77]:
```suggestion
  }

```

File: examples/web/src/__tests__/apollo-graphql-queries-mutations.test.jsx [hunk 3]
```diff
@@ -69,27 +72,87 @@ function MovieList({ movies }) {
...
-    {
-      _id: 3,
-      title: "Dunkirk",
-      year: 2017,
-      runtime: 106,
-    },
-  ];
+  const app = new Realm.App(GRAPHQL_APP_ID);
+  // for some reason these tests running multiple times w/o this 🤷.
+  // my guess is it has something to do with the CRA testing framework
```
Comment (cbush) [inline examples/web/src/__tests__/apollo-graphql-queries-mutations.test.jsx:122]: Just a guess but does the scoping of your calls to beforeAll/afterAll matter? i.e. what if you put them outside of the describe?

**Review by cbush — APPROVED** [review]

File: examples/web/src/__tests__/apollo-graphql-queries-mutations.test.jsx (hunk 3 above)
Comment (mongodben) [reply examples/web/src/__tests__/apollo-graphql-queries-mutations.test.jsx:122]: i put em here while debugging the issue talked about here. i think this comment is no longer true. forgot about. will try to refactor w/o this.

Comment (mongodben) [reply examples/web/src/__tests__/apollo-graphql-queries-mutations.test.jsx]: yeah it's no longer tru. fixed in [7ef6120](https://github.com/mongodb/docs-realm/pull/2663/commits/7ef61200c5115659c7e083464ab67e02ff79a0f3)

**Pull request merged**